src/components/Board/Board.component.js

```diff
@@ -104,7 +107,71 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
```

Comment:
Since `runLiveHelp` and `runLiveUnlock` are boolean variables, it is best practice to prefix them with `should`.
This will help with readability.

Example:
`shouldRunLiveHelp: false`

Reply:
I added the prefix in both variables!